### ISSUE TITLE
Browser keeps opened even after .kill()

### ISSUE THREAD

**Issue by sandro-salles** [open-wa/wa-automate-nodejs#557]
On api/Client.js

```
public async kill() {
    console.log('Shutting Down');
    try{
      if (this.page && !this.page.isClosed()) await this.page.close();
      **if (this.page && !this.page.isClosed() && this.page.browser) await this.page.browser().close();**
    } catch(error){}
    return true;
  }
```
Browser never gets closed because of **!this.page.isClosed()** check... 

Regards

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i`);
exec(`npm run release-ci`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [1.9.75](https://github.com/open-wa/wa-automate-nodejs/compare/1.9.74...1.9.75) (2020-06-17)

- build(deps): bump puppeteer from 3.3.0 to 4.0.0 [`#555`](https://github.com/open-wa/wa-automate-nodejs/pull/555)
- Fix #557 [`#557`](https://github.com/open-wa/wa-automate-nodejs/issues/557)
- Release 1.9.75 [`4cd4214`](https://github.com/open-wa/wa-automate-nodejs/commit/4cd4214c9127e5768b5c7ce2bb5fb2062b8b7670)
- Feature: Added WAPI.js level license mechanism for non-open-wa based projects. #556 [`9696a20`](https://github.com/open-wa/wa-automate-nodejs/commit/9696a2063559fe07fff39d02a5ffd472f94da0d0)
